skimage/morphology/watershed.py

```diff
@@ -156,6 +156,9 @@ def watershed(image, markers, connectivity=1, offset=None, mask=None,
     compactness : float, optional
         Use compact watershed [3]_ with given compactness parameter.
         Higher values result in more regularly-shaped watershed basins.
+    watershed_line: bool, optional (default=False)
```

Comment:
Please, remove `(default=False)`. We mainly use `%param%: %type%[, optional]` all over the package.